bootstrap/test_bootstrap.py

```diff
@@ -31,6 +31,15 @@
             }
         },
         "privileged": true
+    },
+    "ttyd": {
+        "tag": "master",
+        "image": "bluerobotics/companion-ttyd",
```

Comment:
A quick question, how the test is working if `bluerobotics/companion-ttyd` docker does not exist in `hub.docker` ?

Reply:
![image](https://user-images.githubusercontent.com/4013804/149573767-2cc7f27a-5ac3-4327-84e2-497df11b26cf.png)

Reply:
:facepalm:, I always forget that